bin/serve.js

```diff
@@ -154,29 +153,58 @@ const startEndpoint = (endpoint, config) => {
 };
 
 const loadConfig = async (cwd, entry) => {
-	const paths = {
-		'serve.json': null,
-		'now.json': 'static',
-		'package.json': 'now.static'
-	};
+	const files = [
+		'serve.json',
+		'now.json',
+		'package.json'
+	];
 
 	const config = {};
 
-	for (const file of Object.keys(paths)) {
+	for (const file of files) {
 		const location = path.join(entry, file);
-		const prop = paths[file];
+		let content = null;
+
+		try {
+			content = await readFile(location, 'utf8');
```

Comment:
`await` in a loop seems bad?

Reply:
Talked this out via DM. Usually a great idea, but we want to stop the loop immediatley once we notice that a file is readable.

But at the same time allow `readFile` to error in order to cover files that don't exist (which shouldn't make us loose the rest of the values, which is what `Promise.all` would do, e.g).